packages/lib-classifier/test/factories/tasks/MultipleChoiceTaskFactory.js

```diff
@@ -2,9 +2,9 @@ import { Factory } from 'rosie'
 
 export default new Factory()
   .attr('answers', [
-    { label: 'red', _key: Math.random() },
-    { label: 'blue', _key: Math.random() },
-    { labe: 'green', _key: Math.random() }
+    { label: 'red', _key: 1 },
+    { label: 'blue', _key: 2 },
+    { label: 'green', _key: 3 }
```

Comment:
PFE uses `Math.random()` for these keys, so I'd like to use that here so the factory mocks are like what the actual tasks are (or as close as we can mock): 

https://github.com/zooniverse/Panoptes-Front-End/blob/e52b9f39a667f5e1bf4c235622e92f1a5537e2df/app/classifier/tasks/generic-editor.cjsx#L104

Reply:
Why is this model storing these React keys anyway? There are no keys stored on the single choice task answers. In the view code for the multiple choice task, the key for each answer is `${task.taskKey}_${index}`

Reply:
NB. underscored properties in PFE resources are ignored when saving to Panoptes.